docker-compose.yml

```diff
@@ -4,49 +4,109 @@ services:
         image: zquestz/bitcoin-unlimited
         container_name: bitcoind-regtest
         ports:
-          - "18332:18332"
+          - "18443:18443"
           - "28332:28332"
           - "50001:50001"
         volumes:
-          - ./data:/data
-          - ./bitcoin.conf:/data/bitcoin.conf
-          - ./bitcoin.conf:/root/.bitcoin/bitcoin.conf
+          - ./data/bitcoind:/data
+          - ./configs/bitcoin-unlimited.conf:/data/bitcoin.conf
+          - ./configs/bitcoin-unlimited.conf:/root/.bitcoin/bitcoin.conf
         entrypoint: "/entrypoint.sh"
         command: bitcoind
         networks:
           - bitcoin-bridging
 
-    nginx-regtest:
-        image: nginx:latest
-        container_name: nginx-regtest
+    # bitcoind-regtest:
+    #     image: zquestz/bitcoin-cash-node
+    #     container_name: bitcoind-regtest
+    #     ports:
+    #       - "18443:18443"
+    #       - "28332:28332"
+    #     volumes:
+    #       - ./data/bitcoind:/data
+    #       - ./configs/bitcoin.conf:/data/bitcoin.conf
+    #       - ./configs/bitcoin.conf:/root/.bitcoin/bitcoin.conf
+    #     entrypoint: "/entrypoint.sh"
+    #     command: bitcoind
+    #     networks:
+    #       - bitcoin-bridging
+
+    # fulcrum:
```

Comment:
What is this do?

Reply:
This is the Bitcoin Cash Node and Fulcrum indexer (as opposed to the Bitcoin Unlimited node that we currently use). Unfortunately it has the "mainnet" bug currently and I'm still figuring that out. Continue using BU for now.